Sources/NIOWebSocket/Base64.swift

```diff
@@ -18,6 +18,7 @@
 extension String {
 
   /// Base64 encode a collection of UInt8 to a string, without the use of Foundation.
+  @usableFromInline
```

Comment:
Oh sorry, can we make this `@inlinable` too?

Reply:
Done ✅ but does it need to be `@inlinable` to be specialized? We call this method in `randomRequestKey(using:)` and the generic type (`ByteBufferView`) is known inside the module.

Reply:
Yes it does.

The mental model to apply for when `@inlinable` will be needed is not whether the call-site _as written_ is inside the module, but whether the call-site _as compiled_ is inside the module. In this instance, `randomRequestKey(using:)` is `@inlinable`. When compiled and specialized, the effective call site of this code is inside the specialized code, generated by the caller. As a result the _effective_ call-site is in user code, not inside the module.

In generic situations like this you tend to have to add inlinables the whole way down the stack.